app/controllers/assessment/gradings_controller.rb

```diff
@@ -183,6 +194,37 @@ def show
 
   private
 
+  # Compare answers against attempts, modulo case and punctuation
+  def auto_grading_equal(answer, attempt)
+    answer = answer.gsub(/[^\w\d\s]/,'')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.